test/unit/middlewares/authorizeBot.test.js

```diff
@@ -0,0 +1,52 @@
+const authorizeBot = require("../../../middlewares/authorizeBot");
+const sinon = require("sinon");
+const expect = require("chai").expect;
+const botVerifcation = require("../../../services/botVerificationService");
+
+describe("Check authorization of bot", function (done) {
+  it("return false when token is invalid", function () {
+    const request = {
+      headers: "Bot BAD.JWT.TOKEN",
+    };
+
+    const response = {
+      statusCode: 401,
+      error: "Unauthorized",
+      message: "Unauthorized Bot",
+    };
+
+    const nextSpy = sinon.spy();
+    authorizeBot(request, response, nextSpy);
+    expect(nextSpy.calledOnce).to.be.equal(false);
+  });
+
+  it("return false when header is not present", function () {
+    const request = {};
+
+    const response = {
+      statusCode: 400,
+      error: "Invalid Request",
+      message: "Invalid Request",
+    };
+
+    const nextSpy = sinon.spy();
+    authorizeBot(request, response, nextSpy);
+    expect(nextSpy.calledOnce).to.be.equal(false);
+  });
+
+  it("return true when token is valid", function () {
+    const jwtToken = botVerifcation.generateToken();
+
+    const request = {
+      headers: {
+        authorization: `Bot ${jwtToken}`,
```

Comment:
I think we should be using the known authentication schemes here @RitikJaiswal75 
Check here : https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes

Reply:
We can use `Bearer`

Reply:
I am ok with anything here, I told bot as discord API's use `bot` 
Can you ask for others opinions as well?

Reply:
Yeah sure